service-base/src/main/java/org/eclipse/hono/service/tenant/TenantHttpEndpoint.java

```diff
@@ -123,11 +134,12 @@ private void addTenant(final RoutingContext ctx) {
 
         final String tenantId = getTenantIdFromContext(ctx);
 
-        final String location = String.format("/%s/%s", TenantConstants.TENANT_ENDPOINT, tenantId);
+        final String location = String.format("/%s/", TenantConstants.TENANT_ENDPOINT);
```

Comment:
use *getName* instead of `TenantConstants.TENANT_ENDPOINT` when creating *Location* value...